packages/ui/src/components/navbar-skeleton/item.tsx

```diff
@@ -19,7 +19,7 @@ export function Item({ icon, text, description, active, submenuItem, className }
       <div
         className={cn(
           'group relative grid cursor-pointer select-none grid-cols-[auto_1fr] items-center gap-3 pb-[0.6875rem] pt-[0.5625rem] py-2 px-3 rounded-md',
-          { 'bg-background-4': active },
+          // { 'bg-background-4': active },
```

Comment:
does the navbar work fine after this change? i see a change in a shared ocmponent